src/main/java/org/mifos/processor/bulk/camel/routes/InitSubBatchRoute.java

```diff
@@ -115,7 +115,6 @@ public void configure() throws Exception {
                     PaymentModeMapping mapping = paymentModeConfiguration.getByMode(paymentMode);
 
                     String tenantName = exchange.getProperty(TENANT_NAME, String.class);
-                    tenantName = mapping.getDebulkingDfspid() == null ? tenantName : mapping.getDebulkingDfspid();
```

Comment:
why these lines are removed?

Reply:
It was not used anywhere. 
Dfspid been used in line 120 directly                     
variables.put(DEBULKINGDFSPID, mapping.getDebulkingDfspid() == null ? tenantName : mapping.getDebulkingDfspid());